GameData/SpaceDust/Definitions/LqdCO2.cfg

```diff
@@ -0,0 +1,194 @@
+// Defines Carbon Dioxide definitions for stock planets
+
+// KERBIN
+// =============
+SPACEDUST_RESOURCE
+{
+	resourceName = LqdCO2
+	body = Kerbin
+
+	RESOURCEBAND
+	{
+		name = kerbinAtmo
+		title = #LOC_SpaceDust_Band_Atmosphere
+
+		// Discoverability Data
+		alwaysDiscovered = true
+		alwaysIdentified = true
+
+		// Abundance Data
+		minAbundance = 0.01 // X% / 80 (guess-work by JadeOfMaar)
```

Comment:
This is in t/m3.  It should be something like 0.000048 if Kerbin is like earth (1.2 kg/m3 of air, 4% CO2, *0.001kg -> tons)